media/Config/index.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// before building DOM, this function guide which tool will be built
+// you can find makeSelectTagForImport and buildOptionDom in buildDom.js
+const showOptions = function(event) {
+  emptyOptionBox(false);
+  event.target.classList.add('selected');
+  switch (event.target.id) {
+    case 'import': {
+      changeCommonTags(oneImport);
+
+      const select = makeSelectTagForImport();
+
+      if (oneImport.use === false) {
+        select.disabled = true;
+      } else {
+        select.disabled = false;
+      }
+
+      const spanTagForSelect = document.querySelector('#spanTagForSelect');
+      spanTagForSelect.appendChild(select);
+
+      // if framework like tensorflow has already been chosen, then bring it
+      let chosenOptionIndex = -1;
+      for (let i = 0; i < oneImport.options.length; i++) {
+        if (oneImport.options[i].optionValue === true) {
+          chosenOptionIndex = i;
+          break;
+        }
+      }
+      if (chosenOptionIndex !== -1) {
+        select.options[chosenOptionIndex + 1].selected = true;
```

Comment:
why add add 1 as `chosenOptionIndex + 1` ?

Reply:
because `oneImportOption` and `select.options` are different.
```javascript
oneImportoption = [oneImportBcq, oneImportOnnx, oneImportTf, oneImportTflite]
oneImportoption = [beforeDecision, oneImportBcq, oneImportOnnx, oneImportTf, oneImportTflite]
```
I think select tag has default option for choosing nothing.
So i add beforeDecision and this makes `chosenOptionIndex + 1`

Reply:
Adding a comment above this line may help understanding the code better
```js
// select.options has 'beforeDecision' at index 0
```